apis/visibility/v1alpha1/openapi/zz_generated.openapi.go

```diff
@@ -2825,7 +2824,7 @@ func schema_kueue_apis_visibility_v1alpha1_PendingWorkload(ref common.ReferenceC
 					},
 					"priority": {
 						SchemaProps: spec.SchemaProps{
-							Description: "Priority indicates the workload's priority",
+							Description: "Priority indicates the workload's priority asaaaa",
```

Comment:
```suggestion
							Description: "Priority indicates the workload's priority",
```
?

Reply:
Sorry, I left a note for myself and pushed it!

Reply:
fix: https://github.com/kubernetes-sigs/kueue/pull/2528/commits/2db6c62daeae6a417cd4a85fd0d98ef9104f7d63